Source/WebCore/rendering/svg/SVGBoundingBoxComputation.h

```diff
@@ -68,7 +69,10 @@ class SVGBoundingBoxComputation {
 
     static LayoutRect computeVisualOverflowRect(const RenderLayerModelObject& renderer)
     {
-        auto repaintBoundingBoxWithoutTransformations = computeDecoratedBoundingBox(renderer, repaintBoundingBoxDecoration | DecorationOption::IncludeOutline | DecorationOption::IgnoreTransformations);
+        DecorationOptions options = repaintBoundingBoxDecoration | DecorationOption::IncludeOutline | DecorationOption::IgnoreTransformations;
+        if (renderer.isRenderSVGContainer())
```

Comment:
I think we prefer is<RenderSVGContainer>.

Reply:
Sure, will fix, thanks for the review!